python/pyspark/sql/dataframe.py

```diff
@@ -2089,6 +2089,10 @@ def toPandas(self):
 
         .. note:: Usage with spark.sql.execution.arrow.pyspark.enabled=True is experimental.
 
+        .. note:: When Arrow optimization is enabled, bytes and strings in Python 2 are considered
+            identically with Python 2 itself whereas PySpark takes them as strings. When
```

Comment:
`strings in Python 2 are considered identically as bytes with Python 2 itself...`? Current wording sounds a little ambiguous.

Reply:
Hmhmm .. let me try to make it better.